docs/man/docker.1.md

```diff
@@ -292,79 +292,192 @@ inside it)
 
 # STORAGE DRIVER OPTIONS
 
-Options to storage backend can be specified with **--storage-opt** flags. The
-only backend which currently takes options is *devicemapper*. Therefore use these
+Docker uses storage backends (or "graphdriver" in Docker internals) to create
+writable containers from images.  Many of these backends use operating system
+level technologies and can be configured.
+
+Specify options to the storage backend with **--storage-opt** flags. The only
+backend that currently takes options is *devicemapper*. Therefore use these
 flags with **-s=**devicemapper.
 
+Specifically for devicemapper, the default is a "loopback" model which
+requires no pre-configuration, but is extremely inefficient.  Do not
+use it in production.
+
+To make the best use of Docker with the devicemapper backend,
+you must have a recent version of `lvm2`.  Use `lvm` to create a thin
```

Comment:
The CLI is always called `lvm`, `lvm2` just happens to be the package name at least on Fedora.  It's probably saner to just say `lvm` everywhere and people can figure out the package name.  Will fix.